src/chrome/chromeDebugAdapter.ts

```diff
@@ -376,6 +384,26 @@ export abstract class ChromeDebugAdapter implements IDebugAdapter {
         this._chromeConnection.onClose(() => this.terminateSession('websocket closed'));
     }
 
+    private runAndMeasureProcessingTime(notificationName: string, procedure: () => Promise<void>): Promise<void> {
+        const startTime = Date.now();
+        const startTimeMark = process.hrtime();
+        let properties: IExecutionResultTelemetryProperties = {
+            startTime: startTime.toString()
+        };
+
+        return procedure().then(() => {
```

Comment:
I personally find async/await code easier to read and reason than using promises

Reply:
Agreed and changed